models/api.models.js

```diff
@@ -43,3 +43,26 @@ exports.fetchArticles = () => {
   });
 };
 
+
+exports.fetchCommentsByArticleId = (article_id) => {
+  const queryStr = `
+    SELECT 
+      comment_id, 
+      votes, 
+      created_at, 
+      author, 
+      body, 
+      article_id
+    FROM comments
+    WHERE article_id = $1
+    ORDER BY created_at DESC;
+  `;
+
+  return db.query(queryStr, [article_id]).then(({ rows }) => {
+    if(rows.length === 0){
```

Comment:
think about when you read an article on bbc news - if the article has no comments, you dont want to see an error just because nobody has commented on it right? An error will prevent us from loading the page and reading the article - in which case no new article would ever receive a comment because we'd never be able to read it in the first place! So you want to say the request for the comments was successful - you just didn't get any comments back - so we need another 200 test where there's no comments!

However, you will now have to consider how else to pass your 404 test you've written for non-existent article id